addons/website_event/views/event_templates.xml

```diff
@@ -11,710 +11,775 @@
+            <t t-call="website_event.event_cover">
+                <t t-set="use_filters" t-value="True"/>
+                <t t-set="use_size" t-value="True"/>
+                <t t-set="use_text_size" t-value="True"/>
+                <t t-set="use_text_align" t-value="True"/>
+                <div class="container d-flex flex-column flex-grow-1 justify-content-around">
+                    <div class="o_wevent_event_title o_wevent_font_cover_regular">
+                        <span t-if="event.is_participating" class="badge badge-success o_wevent_badge">Registered</span>
+                        <h1 t-field="event.name" class="o_wevent_event_name" itemprop="name" placeholder="Event Title"/>
+                        <h2 t-field="event.subtitle" class="o_wevent_event_subtitle" placeholder="Event Subtitle"/>
+                    </div>
                 </div>
-            </div>
+                <div class="container mb-5">
+                    <t t-call="website_event.registration_template"/>
+                </div>
+            </t>
+            <t t-raw="0"/>
         </div>
     </t>
 </template>
 
-<template id="event_description_full">
+<!-- Event - Description -->
+<template id="event_description_full" name="Event Description">
     <t t-call="website_event.event_details">
-        <div class="col">
-            <div t-if="event.state == 'done' or not registrable" class="alert alert-info" role="status">Registration for this event is now closed.</div>
-            <t t-if="event.state != 'done' and registrable" t-call="website_event.registration_template"/>
-            <div class="clearfix"/>
-            <hr/>
-            <div itemprop="description" t-field="event.description"></div>
-            <div class="clearfix"/>
-            <ul class="list-unstyled" id="comment">
-                <li t-foreach="event.website_message_ids" t-as="comment" class="media mt-3">
-                    <div class="media-body">
-                        <t t-call="website.publish_management">
-                            <t t-set="object" t-value="comment"/>
-                            <t t-set="publish_edit" t-value="True"/>
-                        </t>
-                        <t t-raw="comment.body"/>
-                        <small class="float-right muted text-right">
-                            <div t-field="comment.author_id"/>
-                            <div t-field="comment.date" t-options='{"hide_seconds":"True"}'/>
-                        </small>
+        <section class="bg-200">
+            <div class="container">
+                <div class="row no-gutters mt-n5 pb-5">
+                    <!-- Description -->
+                    <div id="o_wevent_event_main_col" class="col-lg-8 bg-white rounded-left p-5 shadow-sm">
+                        <div t-field="event.description" itemprop="description"/>
+                        <!-- TODO - Comments - Is it dead code? -->
```

Comment:
@qsm-odoo Is this working? Never saw any comments on events.

Reply:
Hum... In fact I discovered a new feature here :') Go the event in the backend, send a message in the chatter and the message will appear thanks to this section (and for each message published this way you can decide if you publish it or not... even though it is published by default o_O).

Well I have doubt about the feature (but no doubt it is badly displayed in the current event app) but if you could keep it and just make it nicer I guess...

Reply:
@qsm-odoo O_o 
Okay. I will make it nicer.